ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/GSM/Model.hs

```diff
@@ -223,15 +213,14 @@ transition model cmd resp = fixupModelState cmd $ case (cmd, resp) of
         mCandidates = cands
       ,
         mClock = clk
-      ,
-        mIdlers = idlers
       ,
         mSelection = Selection b s
       } = model
 
     model' = model { mPrev = WhetherPrevTimePasses False }
 
-    plusC (Candidate x) (Candidate y) = Candidate (x + y)
+    plusC x (Just (PeerState (Candidate y) i)) = Just (PeerState (Candidate (x + y)) i)
```

Comment:
FTR: This fixes the test failures (but see the comment about reverting the changes to this file):
```suggestion
    plusC x (Just (PeerState (Candidate y) _)) = Just (PeerState (Candidate (x + y)) (Idling False))
```